pycargoebuild/ebuild.py

```diff
@@ -211,6 +248,7 @@ def get_ebuild(pkg_meta: PackageMetadata,
                crate_license: bool = True,
                crate_tarball: typing.Optional[Path] = None,
                license_overrides: typing.Dict[str, str] = {},
+               features: bool,
```

Comment:
```suggestion
               use_features: bool,
```

As otherwise it gets confusing when you're referring to the switch, and when to the actual features (yes, other bools in this function would also use a rename).